apix-impl/61/overrides.gradle

```diff
@@ -3,4 +3,5 @@ description = "Xenit API-X implementation Alfresco 6.1"
 ext {
     alfresco_version = alfresco_61_version
     alfresco_dm_version = alfresco_61_dm_version
+    alfresco_min_version = "6.1"
```

Comment:
@kpiot123 could you add the comment here `Not setting max version to make it easier to test on next Alfresco version (only for last version)`, please? I will already approve. No need to assign back to me. After adding this, you may merge.